bundles/org.openhab.binding.mqtt.homeassistant/README.md

```diff
@@ -109,6 +109,18 @@ If a device has multiple device triggers for the same subtype (the particular bu
 | direction       | String  | R/W | `forward` or `backward`                                   |
 | json-attributes | String  | RO  | Additional attributes, as a serialized JSON string.       |
 
+### [Humidifier](https://www.home-assistant.io/integrations/humidifier.mqtt/)
+
+| Channel ID       | Type                 | R/W | Description                                                                              |
+|------------------|----------------------|-----|------------------------------------------------------------------------------------------|
+| state            | Switch               | R/W | If the humidifier should be on or off.                                                   |
+| action           | String               | RO  | What the humidifier is actively doing. One of `off`, `humidifying`, `drying`, or `idle`. |
+| mode             | String               | R/W | Inspect the state description for valid values.                                          |
```

Comment:
sure? Otherwise it is just on off. 
```suggestion
| mode             | String               | R/W | Inspect the `action` description for valid values.                                          |
```

Reply:
No, I mean https://github.com/openhab/openhab-core/blob/4.3.0.M4/bundles/org.openhab.core/src/main/java/org/openhab/core/types/StateDescription.java, not the description of the `state` channel just above this. I use this same verbiage in a few other channels in the README. Any suggestions on how to make it clearer what I'm referring to?

Reply:
Not at the moment, will need to come back to this documentation. We can do that after the weekend